nextflow_schema.json

```
@@ -454,6 +458,53 @@
                     "hidden": true
                 }
             }
+        },
+        "nf_gpt_parameters": {
+            "title": "nf_gpt_parameters",
+            "type": "object",
+            "description": "Contains parameters to control nf-gpt plugin calling.",
+            "default": "",
+            "properties": {
+                "gpt_drugz_gene_amount": {
+                    "type": "integer",
+                    "default": 400,
+                    "description": "Number of top genes to be selected from drugZ."
+                },
+                "gpt_drugz_question": {
+                    "type": "string",
+                    "default": "Which of the following genes have historically shown records of enhancing drug activitys upon CRISPR knockout? For each positive gene write a paragraph explaining the findings in detail.",
+                    "description": "Question parsed with drugZ data to gpt."
```

```suggestion
                    "description": "Prompt to parse drugZ data with nf-gpt."
```

---

or another option
```suggestion
                    "description": "Prompt passed to nf-gpt to parse drugZ data."
```
You can choose or adapt any of my sentences, but please make it consistent through all the parameters, thanks!